UGW:
	Need a generic solution for build directory path legnths more than 128 characters.
	Awaiting owrt community reponse.

diff --git a/tools/quilt/patches/002-fix-bash-path.patch b/tools/quilt/patches/002-fix-bash-path.patch
new file mode 100755
--- /dev/null
+++ b/tools/quilt/patches/002-fix-bash-path.patch
@@ -0,0 +1,12 @@
+FIXME: Need a generic solution for build directory path legnths more than 128 characters. Awaiting owrt community reponse.
+
+--- a/configure.ac	2015-09-28 23:10:53.026959372 +0530
++++ b/configure.ac	2015-09-28 23:11:59.757143677 +0530
+@@ -23,6 +23,7 @@
+ unset BASH  # bash sets this itself!
+ 
+ QUILT_COMPAT_PROG_PATH(BASH, bash)
++BASH="/bin/bash"
+ 
+ # It would be nice not to have to use backticks, but too many retarded sh
+ # implementations still don't support $( )
